app/controllers/admin/users_controller.rb

```diff
@@ -2,7 +2,13 @@ class Admin::UsersController < Admin::BaseController
 
   def index
     authorize(User)
-    @users = User.all.page(params[:page])
+    @users = User.member.page(params[:page])
+  end
+
+  def index_admins
+    authorize(User)
+    @users = User.admin.page(params[:page])
+    render :index
   end
 
```

Comment:
I think we should make use of Pundit `policy_scope` for `index` and `index_admins`:

```ruby
def index
  @users = policy_scope(User.member).page(params[:page])
end

def index_admins
  @users = policy_scope(User.admin).page(params[:page])
end
```

[MINOR][PP] I don't like `member` and `admin` as names for scopes, well maybe this is just me but it seems to return only one member or admin, perhaps pluralising would make it better. But that's my preference though, I'm fine with that if other developers disagree.